plugins/plugin-chart-choropleth-map/src/maps/index.ts

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-webpack-loader-syntax, import/no-unresolved */
+import { keyBy } from 'lodash/fp';
+// @ts-ignore
+import usa from 'file-loader!./usa.topojson';
+// @ts-ignore
+import world from 'file-loader!./world.topojson';
+import { RawMapMetadata } from '../types';
+
+const maps: RawMapMetadata[] = [
+  {
+    key: 'usa',
+    name: 'USA',
+    type: 'topojson',
+    url: usa,
+    keyField: 'properties.STATE',
+    projection: 'Albers',
+  },
+  {
+    key: 'world',
+    name: 'World Map',
+    type: 'topojson',
+    url: world,
+    keyField: 'id',
+    projection: 'Equirectangular',
+    rotate: [-9, 0, 0],
+  },
+];
```

Comment:
How about

```ts
export const maps = {
  usa: {
    ...
  },
  world: {
    ...
  }
};
export const mapChoices = Object.keys(maps); // or manually curated list
```

You get duplicate key checks for free.

Reply:
👍